jobs/ShareMethod.groovy

```diff
@@ -45,14 +45,68 @@ def occupyAvailableLockedResource(String label_name, ArrayList<String> used_reso
     }
 }
 
-def buildAndPublish(){
-    // retry times for package build and images build to avoid failing caused by network
+def buildPackage(String repo_dir){
+    // retry times for package build to avoid failing caused by network
     int retry_times = 3
     stage("Packages Build"){
         retry(retry_times){
-            load("jobs/build_debian/build_debian.groovy")
+            load(repo_dir + "/jobs/build_debian/build_debian.groovy")
         }
     }
+}
+
+def buildImages(String repo_dir){
+    // retry times for images build to avoid failing caused by network
+    int retry_times = 3
+    stage("Images Build"){
+        parallel 'vagrant build':{
+            retry(retry_times){
+                load(repo_dir + "/jobs/build_vagrant/build_vagrant.groovy")
```

Comment:
just a question, can we add some thing between different retry times, to tell people it's retrying and it's No.i iteration ?